<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraChooser</class>
 <widget class="QDialog" name="CameraChooser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>209</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
   </font>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ActionsContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Camera Chooser</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res.qrc">
    <normaloff>:/image/EZCAP_Icon.ico</normaloff>:/image/EZCAP_Icon.ico</iconset>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>115</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="coBox_cameraChooser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>321</width>
     <height>26</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <item>
    <property name="text">
     <string>No Camera</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="okBtn_cameraChooser">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>145</y>
     <width>75</width>
     <height>26</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn_cameraChooser">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>145</y>
     <width>75</width>
     <height>26</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label2_cameraChooser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>175</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Click the logo to learn more about QHYCCD.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label1_cameraChooser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Select the type of camera you have.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logo_cameraChooser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>135</y>
     <width>131</width>
     <height>40</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/qhyccd.png);</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_readmode">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>241</width>
     <height>22</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_readmode">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>readmode</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
